src/Miki/Modules/GeneralModule.cs

```diff
@@ -538,24 +538,23 @@ public async Task InviteAsync(IContext e)
         [Command("ping", "lag")]
         public Task PingAsync(IContext e)
         {
-            return new LocalizedEmbedBuilder(e.GetLocale())
-                .WithTitle(new RawResource("Ping"))
-                .WithDescription("ping_placeholder")
-                .Build()
+            var locale = e.GetLocale();
+            return new EmbedBuilder()
```

Comment:
LocalizedEmbedBuilder has been a failed experiment and will be removed in the next iteration of Miki.Framework

Reply:
I prefer the e.GetLocale() method :)